src/optimize/dynamic_dll_plugin/dynamic_dll_plugin.js

```diff
@@ -456,12 +291,43 @@ export class DynamicDllPlugin {
     return stubModule;
   }
 
+  async areMaxCompilationsPerformed() {
```

Comment:
For functions that are just designed to check a condition and throw if there is a problem I'd normally name them `assertXYZ`, so maybe `assertMaxCompilations()`?

Reply:
ofc @spalger . Changed and thanks for the tip 👍